spec/features/stories_spec.rb

```diff
@@ -39,6 +39,49 @@
       end
     end
 
+    context 'when the user save a story' do
+      let!(:story) { create(:story, title: 'Story', project: project, requested_by: user) }
+
+      before do
+        visit project_path(project)
+        wait_spinner
+        wait_page_load
+      end
+
+      it 'highlights story on click to save', js: true do
+        find(story_selector(story)).trigger('click')
+        expect(page).to have_css('.editing')
+        click_on 'Save'
+        expect(page).not_to have_css('.editing')
+        expect(find(story_selector(story))[:style]).to match(/background-color/)
+      end
+    end
+
+    context 'when the user drags a story' do
+      let!(:story) { create(:story, title: 'Story', project: project, requested_by: user) }
+
+      before do
+        visit project_path(project)
+        wait_spinner
+        wait_page_load
+      end
+
+      it 'check if the story is in the correct column', js: true do
```

Comment:
I think this test may not be in the right place, it's not exactly related to dragging stories. Do we need it? If so, can we move it to a more appropriate  `describe` block?